pages/common/assets/sass/_datatable.scss

```diff
@@ -103,14 +103,24 @@
       border-right: 1px solid #ccc;
     }
 
+    .column-two-thirds {
+      > * {
+        margin-left: $gutter-half;
+      }
```

Comment:
Wouldn't it be better to pad the column rather than margin _every_ child?

Reply:
yeah I did have that when it was column three quarters but that isn't a thing, so I switched to this, then changed the column type anyway. tldr: will fix, at some point